xbmc.py

```diff
@@ -308,11 +308,18 @@ def executeJSONRPC(jsonrpccommand):  # NOSONAR
     :rtype: str
 
     See https://codedocs.xyz/xbmc/xbmc/namespace_j_s_o_n_r_p_c.html
-
     """
 
     json_data = json.loads(jsonrpccommand)
     json_method = json_data["method"]
+    json_params = json_data["params"]
+
+    # Implement some methods for real
+    import jsonrpc
+    if json_method.lower() == 'settings.getsettingvalue':
+        return jsonrpc.Settings().GetSettingValue(**json_params)
```

Comment:
Just pass the settings name. Now the jsonrpc.Settings() is dependent on the content of the JSON request parameters. That is a dependency that is not needed at all.

Reply:
This was exactly my goal. I didn't want to parse the `json_params` here and create mappings, but have `GetSettingValue` use the parameters directly based on the naming.